test/test_examples_2d_eulermulti.jl

```diff
@@ -11,35 +11,48 @@ EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "2d")
 @testset "Compressible Euler Multicomponent" begin
   @testset "elixir_eulermulti_shock_bubble.jl" begin
     @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_eulermulti_shock_bubble.jl"),
-      l2   = [6.03088526e-02, 2.88034228e-03, 2.41548900e+01, 7.83393342e-02, 1.91342787e+04],
-      linf = [7.37128632e-01, 1.05064677e-01, 1.99311122e+02, 2.01502716e+00, 1.57477455e+05],
-      tspan = (0.0, 0.0001))
+      l2   = [7.38219722e+01, 8.80169627e-01, 5.79496021e+04, 1.83101743e-01, 1.11794879e-02],
+      linf = [1.96834444e+02, 7.19013908e+00, 1.58095793e+05, 8.19793567e-01, 8.00831673e-02],
```

Comment:
This doesn't look right. An `linf` error on the order of 1e5? What is going on here? If this is really reasonable, I think it should be documented somewhere. The same goes for the other tests with similar issues below.

Reply:
This should be reasonable, since the pressure (Pa) is about 1e5.

Reply:
OK, thanks for clarifying this. I added a comment in 09605f36 to explain this to unsuspecting users like me;-)